app/controllers/cases_controller.rb

```diff
@@ -88,9 +88,8 @@ def destroy
   end
 
   def history
-    unless @this_case.blank? || @this_case.versions.blank?
-      @case_history = @this_case.try(:versions).order(created_at: :asc)
-    end
+    @case_history = @this_case.try(:versions).sort_by(&:created_at).reverse unless
```

Comment:
Why did you change this back to using `.sort_by`?  I just merged the change to do the reverse at the database level.

Reply:
fixed, it